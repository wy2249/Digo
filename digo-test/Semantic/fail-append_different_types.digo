func digo_main() void {
  a := []int {1,2,3,4}
  var j float
  b := append(a,j)
}
