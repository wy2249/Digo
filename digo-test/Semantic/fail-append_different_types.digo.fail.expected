Fatal error: exception Failure("Built-in Append object and element are not compatible due to different type")
